test/presenters/call_for_evidence_presenter_test.rb

```diff
@@ -0,0 +1,141 @@
+
+    test "presents ways to respond" do
+      example_ways_to_respond = schema_item("open_call_for_evidence_with_participation")["details"]["ways_to_respond"]
+      presented = presented_item("open_call_for_evidence_with_participation")
+
+      assert presented.ways_to_respond?
+      assert_equal example_ways_to_respond["email"], presented.email
+      assert_equal example_ways_to_respond["link_url"], presented.respond_online_url
+      assert_equal example_ways_to_respond["postal_address"], presented.postal_address
+    end
+
+    test "presents a response form when included with email or postal address" do
+      example_ways_to_respond = schema_item("open_call_for_evidence_with_participation")["details"]["ways_to_respond"]
+      presented = presented_item("open_call_for_evidence_with_participation")
+
+      assert presented.response_form?
+      assert_equal example_ways_to_respond["attachment_url"], presented.attachment_url
+
+      example_without_email = schema_item("open_call_for_evidence_with_participation")
+      example_without_email["details"]["ways_to_respond"].delete("email")
+      example_without_email["details"]["ways_to_respond"].delete("postal_address")
+      presented_without_email = presented_item("open_call_for_evidence_with_participation", example_without_email)
+
+      assert_not presented_without_email.response_form?
+    end
+
+    test "does not show ways to respond when call for evidence is closed" do
+      example = schema_item("closed_call_for_evidence")
+      example["details"]["ways_to_respond"] = { "email" => "email@email.com" }
+      presented = presented_item("closed_call_for_evidence", example)
+
+      assert_not presented.ways_to_respond?
+    end
+
+    test "does not show ways to respond when only an attachment url is provided" do
+      example = schema_item("open_call_for_evidence_with_participation")
+      example["details"]["ways_to_respond"].delete("email")
+      example["details"]["ways_to_respond"].delete("postal_address")
+      example["details"]["ways_to_respond"].delete("link_url")
+      presented = presented_item("open_call_for_evidence_with_participation", example)
+
+      assert_not presented.ways_to_respond?
+    end
+
+    test "presents share urls with encoded url and title" do
+      assert_equal "https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.test.gov.uk%2Fgovernment%2Fcall_for_evidence%2Fyouth-vaping-call-for-evidence", presented_item("open_call_for_evidence").share_links[0][:href]
+      assert_equal "https://twitter.com/share?url=https%3A%2F%2Fwww.test.gov.uk%2Fgovernment%2Fcall_for_evidence%2Fyouth-vaping-call-for-evidence&text=Youth%20Vaping", presented_item("open_call_for_evidence").share_links[1][:href]
+    end
+
+    test "presents the single page notification button" do
+      schema = schema_item("open_call_for_evidence")
+      presented = presented_item("open_call_for_evidence", schema)
+
+      assert presented.has_single_page_notifications?
```

Comment:
I've confirmed with Mike, our product owner, that this is expected behaviour for now, as it should match consultation behaviour. If you would recommend we consider migrating both types to the new behaviour then it might be worth us setting up a conversation with him to explore?